modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldValueRenderer.java

```diff
@@ -56,50 +55,45 @@
 	public String render(DDMFormFieldValue ddmFormFieldValue, Locale locale) {
 		Value value = ddmFormFieldValue.getValue();
 
-		return render(value.getString(locale), locale);
+		return render(value.getString(value.getDefaultLocale()), locale);
```

Comment:
@rodrigopaulino why did you change to `value.getDefaultLocale()` when getting the string, but in [line 61](https://github.com/liferay-forms/liferay-portal/pull/1280/files#diff-720a0806d0d1d86d9161878fa05f4753042b94e3bdefc2c89f119244edb075d8R61) you renamed the second parameter to `defaultLocale`? 
`protected String render (String valueString, Locale defaultLocale)`

Reply:
Oh! This was just a test I was doing that I forgot to revert. Thanks for asking.

Reply:
I fixed the `value.getDefaultLocal()` argument, and I changed the name of the parameter to `defaultLocale`, because it will only be used if there's no ThemeDisplay's Locale available.